File: C/URI_judge_1020.c
#include <stdio.h>

int main(){
    int a,m,n,d,t,tm;
    scanf("%d",&n);
    a = n/365;
    n -= a * 365;
    m = n/30;
    n -= m * 30;
    d = n/1;
    printf("%d ano(s)\n",a);
    printf("%d mes(es)\n",m);
    printf("%d dia(s)\n",d);
}

File: Python/URI_judge_1020.py
n = int(input())

a = int(n/365)
n -= a * 365

m = int(n/30)
n -= m * 30

d = int(n/1)

print(f"{a} ano(s)")
print(f"{m} mes(es)")
print(f"{d} dia(s)")